### ISSUE TITLE
Make NoChoiceDiff() etc. a const like noargdiff

### ISSUE THREAD

**Issue by fsaad** [probcomp/Gen.jl#50]

**Comment by fsaad** [comment]
And probably all the Singleton's that appear in `diff.jl`

**Comment by alex-lew** [comment]
I agree, and I think that at some point, the `diff` constants should also be a part of a renaming pass. (This may just be me, but `nochoicediff` and co. sound like we are opting to pass no diff, rather than that we are passing a diff that specifies nothing changed.)

**Comment by marcoct** [comment]
FWIW, the extant closest equivalent values to those for which the ticket was originally created are now `NoChange()` and `UnknownChange()` and there are no constants for these at the moment. But note that there are also non-singleton types: `SetDiff`, `DictDiff`, `VectorDiff`, `IntDiff`, etc.
